CVQM.md

```diff
@@ -0,0 +1,79 @@
+# CVQM Overview
+CVQM performs automated processing on a pair of video files. One contains an original video sequence (e.g., straight from the camera) and the other contains a processed video sequence (e.g., after coding and transmission and decoding). 
+
+All video sequences must be in uncompressed AVI files, in either the UYVY or RGB color space. Original and processed video sequences must contain the same image size and frames per second. CVQM compares the original video sequence to the processed video sequence (i.e., the sequence that has been processed by the video system under test). 
+
+Every original / processed video sequence pair is run through two main steps. First, the requested calibration is run, and results are saved in a file named after the processed video file, with '_calibration.txt' appended. Second, the requested model is run, and results are saved in a file named after the processed video file, with '_model.txt' appended.
+
+Any errors that occur are recorded in a file named after the processed video file, with '_errors.txt' appended. If that file is absent, no errors occurred. Each line on this file will start with a number, indicating error type, and then a string describing the error. 
+Type 1 errors are fatal data input/output issues (e.g., operation cannot continue due to invalid input argument, file read error, or file write error). Type 2 errors are calibration warnings (e.g., still sequence, temporal registration failure, spatial shift beyond search limits). Type 3 errors are non-fatal warnings.
+
+## VQM Model Options
+The following VQM models options are available:
```

Comment:
Is "... models options ..." correct?
Should this be "model options" or "model's options"?

Reply:
changed to VQM software's options